src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -14,8 +14,9 @@
           AfterTargets="Pack"
           Condition="'$(IsPackable)' == 'true'">
 
-    <PropertyGroup >
+    <PropertyGroup>
       <PackageValidationBaselinePath Condition="'$(PackageValidationBaselinePath)' == '' and '$(PackageValidationBaselineVersion)' != ''">$([MSBuild]::NormalizePath('$(NuGetPackageRoot)', '$(PackageValidationBaselineName.ToLower())', '$(PackageValidationBaselineVersion)', '$(PackageValidationBaselineName.ToLower()).$(PackageValidationBaselineVersion).nupkg'))</PackageValidationBaselinePath>
+      <PackageValidationBaselinePath Condition="'$(DisablePackageBaselineValidation)' == 'true'"></PackageValidationBaselinePath>
```

Comment:
This won't work if the path is passed in as a global property. As that property is exposed you want to guard for such a case as well. I think it would make more sense to pass that property to the task as you had it before you removed it.

Aside, is there a reason why the Error task below isn't in the Task so that it can be easily localized?

Reply:
> Aside, is there a reason why the Error task below isn't in the Task so that it can be easily localized?

The purpose was to have a more readable message than this (i.e. without the callstack)
```
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:  ---> System.IO.FileNotFoundException: Could not find file 'C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.1.0.0.nupkg'. [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018: File name: 'C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.1.0.0.nupkg' [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options) in System.Private.CoreLib.dll:token 0x6000122+0xb4 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize) in System.Private.CoreLib.dll:token 0x6000121+0x3e [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at System.IO.Strategies.WindowsFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize) in System.Private.CoreLib.dll:token 0x6005e56+0x5c [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize) in System.Private.CoreLib.dll:token 0x6005db8+0x37 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at System.IO.File.OpenRead(String path) in System.Private.CoreLib.dll:token 0x600598c+0x9 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at NuGet.Packaging.PackageArchiveReader..ctor(String filePath, IFrameworkNameProvider frameworkProvider, IFrameworkCompatibilityProvider compatibilityProvider) in NuGet.Packaging.dll:token 0x6000165+0x2a [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    --- End of inner exception stack trace --- [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at NuGet.Packaging.PackageArchiveReader..ctor(String filePath, IFrameworkNameProvider frameworkProvider, IFrameworkCompatibilityProvider compatibilityProvider) in NuGet.Packaging.dll:token 0x6000165+0x67 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.DotNet.PackageValidation.NupkgParser.CreatePackage(String packagePath, RuntimeGraph runtimeGraph) in C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\NupkgParser.cs:line 26 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.DotNet.PackageValidation.ValidatePackage.ExecuteCore() in C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\ValidatePackage.cs:line 48 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.NET.Build.Tasks.TaskBase.Execute() in C:\git\sdk\src\Tasks\Common\TaskBase.cs:line 38 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() in Microsoft.Build.dll:token 0x60015fc+0x3e [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
C:\git\sdk\src\Compatibility\Microsoft.DotNet.PackageValidation\build\Microsoft.DotNet.PackageValidation.targets(25,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) in Microsoft.Build.dll:token 0x600147b+0x804 [C:\git\sdk\artifacts\tmp\Debug\ValidatePacka---7A559569\PackageValidationTestProject.csproj]
```

Reply:
Got it.

Reply:
I agree, we should pass down the property to the task for the reason listed above.